### DISCUSSION TITLE
Lap comparison on track map

### DISCUSSION THREAD

**Discussion by cecereroberto** [theOehrly/Fast-F1#279] [Q&A]
Hello, my name is Roberto Cecere and I’m an IT engineer who writes for the online newspaper formulaunoanalisitecnica.com. I’m contacting you to ask if it was possible to have free practice or qualyfing simulation of United States Grand Prix. In particular, the overlay lap of the best time of the session with the best time of one of the two Ferrari. It would be an honor for us to have your exclusive and a further opportunity to sponsor your wonderful work. Can I contact you privately? my email is cecere.roberto@gmail.com . Best Regards

Ing. Roberto Cecere

**Comment by borolepratik** [comment]
@cecereroberto feel free to also check out https://www.f1-tempo.com/.

**Reply by cecereroberto** [reply]
I meant something like that

[https://www.reddit.com/r/formula1/comments/xt12da/animated_comparison_of_vers_aborted_lap_and_lecs/]

**Reply by w-i-z-a-r-d** [reply]
these two visuals were made using fastF1
https://twitter.com/F1Techy/status/1527750761412988928?s=20&t=YlxRaS5sF0eWCQt59z24-A
https://twitter.com/F1Techy/status/1527741278917693450?s=20&t=YlxRaS5sF0eWCQt59z24-A
